daemon/api_sideload_n_try.go

```diff
@@ -527,7 +526,7 @@ func readForm(reader *multipart.Reader) (_ *Form, apiErr *apiError) {
 // its path. If the path is not empty then a file was written and it's the
 // caller's responsibility to clean it up (even if the error is non-nil).
 func writeToTempFile(reader io.Reader) (path string, err error) {
-	tmpf, err := ioutil.TempFile(dirs.SnapBlobDir, dirs.LocalInstallBlobTempPrefix)
+	tmpf, err := os.CreateTemp(dirs.SnapBlobDir, dirs.LocalInstallBlobTempPrefix+"*.snap")
```

Comment:
If you're referring to the `.snap` suffix, then that is because the seed writer won't operate on snap files that do not end in `.snap`: https://github.com/snapcore/snapd/blob/master/seed/seedwriter/writer.go?plain=1#L415

If you're talking about the `os.CreateTemp` change, that is because the entire `ioutil` package has been deprecated and the various functions in there put in `io` and `os`.

Reply:
Was referring to the suffix change, but thanks!